base/src/org/compiere/process/DunningPrint.java

```diff
@@ -155,46 +153,37 @@ else if (to.getEMail() == null || to.getEMail().length() == 0)
 				re = new ReportEngine(getCtx(), format, query, info);
 			boolean printed = false;
 			if (isEMailPDF()) {
-				EMail email = client.createEMail(to.getEMail(), null, null);
-				if (!email.isValid()) {
-					addLog (entry.get_ID(), null, null, 
-						"@RequestActionEMailError@ Invalid EMail: " + to);
-					internalError++;
-					continue;
-				}
 				if(text == null) {
 					continue;
 				}
 				text.setUser(to);	//	variable context
 				text.setBPartner(bp);
 				text.setPO(entry);
 				String message = text.getMailText(true);
-				if (text.isHtml()) {
-					email.setMessageHTML(text.getMailHeader(), message);
-				} else {
-					email.setSubject (text.getMailHeader());
-					email.setMessageText (message);
-				}
-				//
+				//	Get instance for notifier
+				DefaultNotifier notifier = (DefaultNotifier) QueueLoader.getInstance().getQueueManager(DefaultNotifier.QUEUETYPE_DefaultNotifier)
+						.withContext(getCtx())
+						.withTransactionName(get_TrxName());
+				//	Send notification to queue
+				notifier
+					.clearMessage()
+					.withApplicationType(DefaultNotifier.DefaultNotificationType_UserDefined)
+					.addRecipient(getAD_User_ID())

```

Comment:
This maybe should be the AD_User_ID of the variable "to", not the AD_User_ID of the running process

Reply:
The user id was corrected, this is loading the correct user id, now I need to test this PR (https://github.com/adempiere/adempiere/pull/3568) for the exception launched to